Sources/Smelter/Onnx_TensorProto+Extensions.swift

```diff
@@ -13,22 +13,21 @@ extension Onnx_TensorProto {
             return self.int32Data.map(Int.init)
         case DataType.int64.rawValue:
             if self.int64Data.count == 0 {
-                return self.rawData.convertToArrayOfType(Int64.self, length: self.length).map(Int.init)
+                return self.rawData.array()
```

Comment:
@eugenebokhan I think here and below: (self.rawData.array() as [T]).map(Int.init)

first - bound data to array of fixed size type, second - map to any other type